doc/source/index.rst.template

```diff
@@ -39,7 +39,7 @@ See the :ref:`overview` for more detail about what's in the library.
 {% endif %}
 
     {% if not single_doc -%}
-    What's New <whatsnew/v0.24.0>
+    whatsnew/v0.24.0
```

Comment:
```suggestion
    What's New in 0.24.0 <whatsnew/v0.24.0>
```

Reply:
You're the one who will have to change the version twice, that's all right with me. :)